Flow.Launcher/Resources/Controls/ExCard.xaml

```diff
@@ -14,7 +14,6 @@
                 x:Name="expanderHeader"
                 Padding="0"
                 BorderThickness="1"
-                IsExpanded="{Binding Mode=TwoWay, Path=IsExpanded}"
```

Comment:
Removed

Reply:
What is this card for?

Reply:
item in setting window. it's custom control included split setting window PR. 
before         
```
<Border Style="{DynamicResource SettingGroupBox}">
                                <ItemsControl Style="{StaticResource SettingGrid}">
                                    <StackPanel Style="{StaticResource TextPanel}">
                                        <TextBlock Style="{DynamicResource SettingTitleLabel}" Text="{DynamicResource hideNotifyIcon}" />
                                        <TextBlock Style="{DynamicResource SettingSubTitleLabel}" Text="{DynamicResource hideNotifyIconToolTip}" />
                                    </StackPanel>
                                    <ui:ToggleSwitch
                                        Grid.Column="2"
                                        FocusVisualMargin="5"
                                        IsOn="{Binding Settings.HideNotifyIcon}"
                                        Style="{DynamicResource SideToggleSwitch}" />
                                </ItemsControl>
                            </Border>
```
After
```
       <cc:Card
                Title="{DynamicResource portableMode}"
                Icon="&#xe88e;"
                Sub="{DynamicResource portableModeToolTIp}">
                <ui:ToggleSwitch IsOn="{Binding PortableMode}" />
            </cc:Card>
```